### PR TITLE
Exposed MG pagerank solver parameters

### LINKED ISSUE #379: [BUG] CHANGELOG.MD 0.8 release date TBD
**Describe the bug**
Changelog does not reflect 0.8 release date.

Instead, it states:
 
> cuGraph 0.8.0 (Date TBD)

**To Reproduce**
Open CHANGELOG.md on 0.9 branch

**Expected behavior**
0.8 release date updated

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by afender**
This is the cugraph part of https://github.com/rapidsai/dask-cugraph/pull/32

Also close #379

### CODE REVIEW

File: python/cugraph/snmg/link_analysis/mg_pagerank_wrapper.pyx [hunk 1]
```diff
@@ -27,7 +27,10 @@ from librmm_cffi import librmm as rmm
 import numpy as np
 
 
-def mg_pagerank(src_ptrs_info, dest_ptrs_info):
+def mg_pagerank(src_ptrs_info,
+                dest_ptrs_info,
+                alpha=0.85,
+                max_iter=30,):
```
Comment (Iroy30) [inline python/cugraph/snmg/link_analysis/mg_pagerank_wrapper.pyx]: the default of max_iter is 30 and dask_cugraph is 50. Could be a confusion as to the default value.

Comment (afender) [reply python/cugraph/snmg/link_analysis/mg_pagerank_wrapper.pyx]: Good catched. Fixed in dask-cugraph https://github.com/rapidsai/dask-cugraph/commit/91fbdf4eef2448f264c33c6c0b5b0e2e7f4bae0e

**Comment by afender** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun test

**Comment by BradReesWork** [comment]
rerun test

**Pull request merged**